WIP-CHANGELOG.md

```diff
@@ -34,10 +34,12 @@
  - Added the new OPA policy - disallowed the latest image tag.
  - Moved `user.alertmanager.group_by` to `prometheus.alertmanagerSpec.groupBy` in `sc-config.yaml`
  - Moved `user.grafana.userGroups` to `user.grafana.oidc.userGroups` in `sc-config.yaml`
+ - Thanos is now enabled by default
```

Comment:
I think this should be more clear (put in a `Release notes` section in the top of the file)

Reply:
Done